airflow/example_dags/example_kubernetes_executor_config.py

```diff
@@ -89,7 +89,6 @@ def test_volume_mount():
 
         # [START task_with_template]
         executor_config_template = {
-            "pod_template_file": os.path.join(AIRFLOW_HOME, "pod_templates/basic_template.yaml"),
```

Comment:
This is used in the docs on https://airflow.apache.org/docs/apache-airflow/stable/executor/kubernetes.html#kubernetesexecutor-architecture

![image](https://user-images.githubusercontent.com/34150/139829123-30594d27-0353-4cb8-b2a4-eef7aa7f37c8.png)

Reply:
Can you update this please @ephraimbuddy